compiler/extccomp.nim

```diff
@@ -763,8 +763,8 @@ template tryExceptOSErrorMessage(conf: ConfigRef; errorPrefix: string = "", body
     if errorPrefix.len > 0:
       rawMessage(conf, errGenerated, errorPrefix & " " & ose.msg & " " & $ose.errorCode)
     else:
-      rawMessage(conf, errGenerated, "execution of an external program failed: '$1'" %
-        (ose.msg & " " & $ose.errorCode))
+      rawMessage(conf, errGenerated, "execution of an external program failed: '$1' error code: $2" %
```

Comment:
Will this change the format of the message that was previously display (before this PR)?

Reply:
Yes. I changed it because the "error code" makes the message clearer.

Reply:
If you want to fix the trailing space, fix the trailing space. Don't make the output even more verbose. If you want to know the exit code, run the program yourself, it's noise for me ("oh no, it failed with error code 5! That's so much worse than error code 1!").

Reply:
The error code was there, I think 'command' returned 5 is less confusing than  'command 5'